gensim/models/utils_any2vec_fast.pyx

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env cython
+# cython: boundscheck=False
+# cython: wraparound=False
+# cython: cdivision=True
+# coding: utf-8
+
+def ft_hash(unicode string):
```

Comment:
why not `cdef` with `nogil`?

Reply:
Both functions need to be called from both Python and Cython, so `cdef` won't work, but `cpdef` should. Will be fixed.

Reply:
`nogil` doesn't work, since the function returns a Python object.